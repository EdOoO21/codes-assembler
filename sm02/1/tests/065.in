-409 -411 53 729 -550 -92 -62 82 -93 128 898 -306 691 309 -107 843 -539 -853 813 148 -44 -460 -597 342 -31 276 423 -907 -474 -769 -674 82 508 758 -723 633 404 687 -745 -720 413 -283 -248 181 336 -250 264 -528 888 784 300 -534 -185
