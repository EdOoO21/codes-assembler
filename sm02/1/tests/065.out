-185
-534
300
784
888
-528
264
-250
336
181
-248
-283
413
-720
-745
687
404
633
-723
758
508
82
-674
-769
-474
-907
423
276
-31
342
-597
-460
-44
148
813
-853
-539
843
-107
309
691
-306
898
128
-93
82
-62
-92
-550
729
53
-411
-409
